Makefile.devel

```diff
@@ -31,6 +31,11 @@ man: third-party-chpldoc-venv FORCE
 man-chpldoc: FORCE
 	cd man && $(MAKE) chpldoc
 
+man-chapel: FORCE
+	cd doc/sphinx && $(MAKE) man-chapel
+	mkdir -p man/man3
+	cp doc/sphinx/build/man/chapel.3 man/man3/chapel.3
+
```

Comment:
Yes - we build a `chpl` man page on [line 28](https://github.com/ben-albrecht/chapel/blob/ea3eb7d9bd5b71f0fb9fa2b289b36caf19fafb1f/Makefile.devel#L28) of this version of the Makefile. This is building a man page that contains the entirety of http://chapel.cray.com/docs/master/. You can think of this as the *language man page* rather than the *compiler man page*. I chose `man-chapel` to follow the convention we used for `man-chpldoc` target.